seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -55,6 +55,9 @@ ethereum,DODO,2,2022-07-24,0x3d01d3959d15fef7be9876f578f87c0c654e2efafb2d5514211
 ethereum,DODO,2,2021-06-03,0x399c234917228e17b4f730a4cd5522d198a7e5f868ba147a8e63676e7d41d17e,66,0x437daacccf7bfafce1ac5828804e2c792242cd16,68.63418,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,33.964561
 ethereum,DODO,2,2021-08-26,0xd534643d2f49ee7c7d0020a4e17e9b9e8d7f921e110779fa5c5bd1865e27cb83,290,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,100000,0x6b175474e89094c44da98b954eedeac495271d0f,99930.038478836639848083
 ethereum,DODO,2,2021-05-14,0xb386cb8c8412be350f24c05360e29f3319a903363e4d34076352bb35b44bc758,62,0xdac17f958d2ee523a2206206994597c13d831ec7,75000,0x57ab1ec28d129707052df4df418d58a2d46d5f51,74754.450749991178425833
+bnb,DODO,0,2021-12-02,0x3d45d93b6359f62a1ec8134f6d8d1bb130c1ead01d8db57244041e24068b1f85,1926,0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c,0.005982,0xe9e7cea3dedca5984780bafc599bd69add087d56,3.729322856407376809
+bnb,DODO,1,2022-05-09,0x3700c79d69a892d2242a66e5975892fbb623fbcdd58cb05d16526356921b877e,42,0xe9e7cea3dedca5984780bafc599bd69add087d56,999.735607882091781117,0x55d398326f99059ff775485246999027b3197955,1000
+bnb,DODO,2_dvm,2022-06-04,0x6e1d453d23a634d5761cee9ec89d829124e2ad79ddb99b91a861aaed5d2344db,134,0x67ee3cb086f8a16f34bee3ca72fad36f7db929e2,2920.79,0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c,1.670916863658532791
```

Comment:
In my last commits I added 3 seeds for DODO_bnb_trades.
According to the model: version `0` is from `aggregator_trades` while [`1`, `2_dvm`] is from `pool_trades`.

BSCSCAN corresponding to the seeds:
- https://bscscan.com/tx/0x3d45d93b6359f62a1ec8134f6d8d1bb130c1ead01d8db57244041e24068b1f85
- https://bscscan.com/tx/0x3700c79d69a892d2242a66e5975892fbb623fbcdd58cb05d16526356921b877e
- https://bscscan.com/tx/0x6e1d453d23a634d5761cee9ec89d829124e2ad79ddb99b91a861aaed5d2344db